### ISSUE TITLE
rubyOffset and line-spacing

### ISSUE THREAD

**Issue by r12a** [w3c/ttml2#252]
10.2.35 tts:rubyOffset
https://www.w3.org/TR/2016/WD-ttml2-20161117/#style-attribute-rubyOffset

How does ruby offset interact with line spacing?  Are we all covered there?

**Comment by nigelmegitt** [comment]
@cconcolato does resolving #373 make the answer to @r12a 's question "yes"?

**Comment by cconcolato** [comment]
@nigelmegitt I don't think so. IIUC, during the meeting with the CSS WG at TPAC it was said that ruby annotations do not contribute to the line height. Also I don't understand what rubyOffset='auto' would mean. #373 does not give enough context. 

Frankly, at this stage, I would be in favor of either deferring rubyOffset to the next version (when CSS has defined it) or of marking it at Risk.

**Comment by nigelmegitt** [comment]
@cconcolato if you don't think we need rubyOffset and you don't intend to implement it, or have insufficient confidence that your implementation will be interoperable (e.g. with CSS), I have no objection to deferring it.

**Comment by r12a** [comment]
Removal seems to void the original question. I have recommended to the i18n WG that we close our tracker for this issue.

**Comment by nigelmegitt** [comment]
Thank you, I'm going to label this as commenter agreed; @r12a if your recommendation to i18n is not accepted please could you let us know? Otherwise we'll assume there's no further action to take.

**Comment by r12a** [comment]
The i18n WG has closed its tracker for this issue.
